compiler/testData/debug/localVariables/constructors/multipleConstructors.kt

```diff
@@ -0,0 +1,38 @@
+
+
+// FILE: test.kt
+open class Base(i: Int)
+
+class Derived(): Base(1) {
+    constructor(p: Int): this() {
+        val a = 1
+    }
+
+    constructor(p1: Int, p2: Int): this()
+}
+
+fun box() {
+    Derived(1)
+    Derived(1, 1)
```

Comment:
I know this is a very minor issue and not even related to the changes in the PR, but in such kind of tests I prefer to always use different values if possible, just to additionally check that we don't mix them up somehow. :) It also makes it a bit easier to read the expectations below. I.e. in this test there's `val a = 1`, so this could be
```
    Derived(2)
    Derived(3, 4)
```

Reply:
No, I agree! I did similar changes on the plug-in side previously because a big aggregate test made 10 assertions with the same constants :sweat_smile:

Reply:
Done :+1: